core/lib/refinery/crud.rb

```diff
@@ -283,7 +283,8 @@ def reorder
             # Based upon https://github.com/matenia/jQuery-Awesome-Nested-Set-Drag-and-Drop
             def update_positions
               previous = nil
-              params[:ul].each do |_, list|
+              # TODO: Find fix because `to_unsafe_h` badly reorder the hash
```

Comment:
Removing the `to_unsafe_h` function here doesn't effect the tests. We'll only get a deprecation warning.